ui/apps/platform/cypress.config.js

```diff
@@ -25,7 +26,7 @@ module.exports = {
 
     e2e: {
         baseUrl: 'https://localhost:3000',
-        specPattern: 'cypress/integration/**/*.test.{js,ts}',
+        specPattern: 'cypress/integration*/**/*.test.{js,ts}',
```

Comment:
I think ideally I'd like to keep this so that we can run e2e tests against standalone ACS UI and the OCP plugin UI separately. I'm OK with keeping this as-is, and making the plugin e2e tests something we run manually for now. Most of my changes in the parent branch were proof-of-concept changes to see what was possible. As long as the variables are exposed to our test jobs and the console is accessible we can work out the details later.
```suggestion
        specPattern: 'cypress/integration/**/*.test.{js,ts}',
```